docs/openapi/schemas.yaml

```diff
@@ -29,6 +29,174 @@ URL:
   type: string
   format: url
   example: 'https://www.example.com'
+SlackMentionConfig:
+  type: object
+  additionalProperties: true
+  properties:
+    slack:
+      type: array
+      items:
+        type: string
+AlertConfig:
+  type: object
+  additionalProperties: true
+  properties:
+    type:
+      description: The type of the alert
+      type: string
+    byOrg:
+      description: Whether the alert should be sent by organization
+      type: boolean
+    mentions:
+      description: The Slack mentions to include in the alert
+      type: array
+      items:
+        $ref: '#/SlackMentionConfig'
+AlertsConfig:
+  type: array
+  items:
+    $ref: '#/AlertConfig'
+SlackConfig:
+  type: object
+  additionalProperties: true
+  properties:
+    channel:
+      description: The Slack channel to send notifications to
+      type: string
+    workspace:
+      description: The Slack workspace to send notifications to
+      type: string
+Config:
+  type: object
+  additionalProperties: true
+  properties:
+    slack:
+      description: Optional. The Slack configuration
+      $ref: '#/SlackConfig'
+    alerts:
+      description: Optional. The Alert configuration
+      $ref: '#/AlertsConfig'
+OrganizationConfig:
+  allOf:
+    - $ref: '#/Config'
+Organization:
+  type: object
+  properties:
+    id:
+      description: The organization ID in uuid format
+      $ref: '#/Id'
+    name:
+      description: The name of the organization
+      type: string
+    imsOrgId:
+      description: Optional. The ID of the Adobe IMS organization
+      type: string
+    config:
+      description: Optional. The configuration of the organization
+      $ref: '#/OrganizationConfig'
+    createdAt:
+      description: The date and time when the organization was created
+      $ref: '#/DateTime'
+    updatedAt:
+      description: The date and time when the organization was last updated
+      $ref: '#/DateTime'
+  example:
+    id: 'o1p2q3r4-s5t6-u7v8-w9x0-yz12x34y56z'
+    name: 'Example Organization'
+    imsOrgId: '1234567890ABCDEF12345678@AdobeOrg'
+    config:
+      slack:
+        channel: '#example-channel'
+        workspace: 'example-workspace'
```

Comment:
Great to have these examples. 

Does the `workspace` property need to include additional details, for example to handle when the Slack workspace is enterprise? (ie. adobe-dx-support.slack.com vs. adobe.enterprise.slack.com)

Reply:
probably not, as the workspace is linked via a provisioned token. @ekremney ?

Reply:
@blefebvre additionally i think we can determine from within the api client whether we're enterprise or not. this distinction has mostly impact on which slack API methods to use (e.g. `admin.conversation.create` vs `conversation.create`)